app/models/topic.rb

```diff
@@ -1060,7 +1060,12 @@ def update_statistics
   end
 
   def update_action_counts
-    update_column(:like_count, Post.where(topic_id: id).sum(:like_count))
+    update_column(
+      :like_count,
+      Post
```

Comment:
Do we need a migration for that?

Reply:
Theoretically, yes, we need one to ensure that all past counts are correct. I did not include one because I imagine it would be very expensive for forums with many topics. I believe I could scope it only to topics which have whisper posts, which would significantly reduce the number of queries.

Reply:
I would also be fine if it was fixed by a "ensure consistency" job.

Reply:
In this case I think a migration is the better solution. :blush: